database/frontend/images.json

```diff
@@ -25,5 +25,20 @@
         "url": "https://kaboompics.com/",
         "category": "frontend",
         "subcategory": "images"
+    },
+    {
+        "name": "Picjumbo",
+        "description": "Download free stock photos, backgrounds and free high-resolution images for personal and commercial use",
```

Comment:
Maybe you can discuss the benefits of this source?

Reply:
Yeah, should be at least 15 words long. thanks chrissy :)